src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -1,32 +1,43 @@
 package mate.academy.hibernate.relations;
 
 import java.util.List;
+import mate.academy.hibernate.relations.dao.ActorDao;
```

Comment:
Ensure that the `hibernate.cfg.xml` file is created and properly configured. This file is crucial for setting up the Hibernate session factory and managing database connections.